modules/openy_gc_shared_content/src/Plugin/Action/UnshareFromVirtualYBase.php

```diff
@@ -17,5 +17,23 @@ class UnshareFromVirtualYBase extends FieldUpdateActionBase {
   protected function getFieldsToUpdate() {
     return ['field_gc_share' => self::GC_SHARE_DEFAULT_VALUE];
   }
+  
+  /**
+   * {@inheritdoc}
+   */
+  public function access($object, AccountInterface $account = NULL, $return_as_object = FALSE) {
+
+    if (!$object->hasField('field_gc_share')) {
```

Comment:
Let's remove `\Drupal::messenger()->addError(t('Error message.'));` line and return `$result->isForbidden()` instead of `FALSE`

Reply:
I have updated the code as per above suggestion, please review.